src/main/groovy/com/rllc/spreadsheet/dao/GoogleSpreadsheetMinisterDAO.groovy

```diff
@@ -0,0 +1,74 @@
+package com.rllc.spreadsheet.dao
+
+import com.google.gdata.client.spreadsheet.SpreadsheetService
+import com.google.gdata.data.spreadsheet.*
+import com.rllc.spreadsheet.domain.Minister
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+/**
+ * Created by Robert on 5/9/15.
+ */
+
+@Component
+class GoogleSpreadsheetMinisterDAO extends AbstractGoogleSpreadsheetDAO implements MinisterDAO {
+
+    private static final Logger logger = LoggerFactory.getLogger(GoogleSpreadsheetMinisterDAO.class)
+
+
+    private static final String MINISTER = "minister"
+
+    @Value("\${google.ministerSpreadsheet}")
+    String googleSpreadsheet
+
+    SpreadsheetService spreadsheetService
+    URL listFeedUrl
+    ListFeed listFeed
+
+
+    public String getGoogleSpreadsheet() {
```

Comment:
groovy lets you access properties directly by doing some AST magic - all properties on a class are enriched with getters and setters.

e.g.
```groovy
def ministerDao = new GoogleSpreadsheetMinisterDAO()
log.info ministerDao.googleSpreadsheet
```